rcl_interfaces/msg/ParameterValue.msg

```diff
@@ -6,8 +6,8 @@
 uint8 type
 
 # "Variant" style storage of the parameter value.
-bool bool_value
-int64 integer_value
-float64 double_value
-string string_value
+bool bool_value False
```

Comment:
Please use `false`(lowercase) to match the [design specification](http://design.ros2.org/articles/interface_definition.html)